legacy/JavaScript/JavaScriptSDK.Module/AppInsightsModule.ts

```diff
@@ -64,7 +64,7 @@ class AppInsightsModule {
     }
 
     private static _download(aiConfig: Microsoft.ApplicationInsights.IConfig) {
-        AppInsightsModule.appInsightsInstance.config = aiConfig;
```

Comment:
While the get accessor will exist for an ES5+ environment we can't use it here as we *could* be running in an ES3 version where the get accessor won't exist.